README.md

```diff
@@ -1,25 +1,52 @@
-# Norman Core SDK
+# Norman Core SDK Overview
 
-A Python Low Level SDK for Norman, providing access to the entire Norman Api.
+The Norman Core SDK is a low-level foundational library that powers all Norman services and underlies the higher-level Norman SDK.
```

Comment:
The Norman Core SDK is the low-level foundational library that powers Norman inter-service communication and underlies the higher-level Norman SDK.